docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -174,27 +173,31 @@ dune --create-account testaccount
 dune --send-action hello test '[bob]' testaccount
 ```
 
-You should see a transaction executed successfully on the first time, and if you try to repeat this command it will 
-fail because that row already exists in the contract's database.
+You should see a transaction executed successfully. If you try to repeat this command, the command 
+fails because the row already exists in the contract's database.
 
 ### Get data from your contract
 
-You just added a row to the contract's database, let's fetch that data from the chain:
+You added a row to the contract's database. 
```

Comment:
Again, sterilization.

Reply:
Simplify language for Hemingway grade level.